x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -773,4 +775,47 @@ export function trainedModelsRoutes(
         }
       })
     );
+
+  /**
+   * @apiGroup TrainedModels
+   *
+   * @api {post} /internal/ml/trained_models/install_elastic_trained_model/:modelId Installs ELSER model
```

Comment:
Nit - should this be `Installs Elastic trained model` - does it support other types of model if and when they become available?